mobile/src/lib/cozy-helper.js

```diff
@@ -83,10 +83,10 @@ export function refreshFolder (dispatch, getState) {
 }
 
 export const onError = (dispatch, getState) => (err) => {
-  if (err.message === clientRevokedMsg) {
+  if (err.message === clientRevokedMsg || err.error === 'code=400, message=Invalid JWT token') {
```

Comment:
it's not the same here https://github.com/cozy/cozy-files-v3/pull/207/files#diff-f96699990a60f265f8d3b7bd25d3e9a6R51 ?